sql/src/main/java/io/crate/expression/reference/sys/check/cluster/LicenseCheck.java

```diff
@@ -27,34 +27,41 @@
 import io.crate.license.LicenseExpiryNotification;
 import io.crate.license.LicenseService;
 import io.crate.settings.SharedSettings;
+import org.elasticsearch.cluster.service.ClusterService;
 import org.elasticsearch.common.inject.Inject;
 import org.elasticsearch.common.inject.Singleton;
 import org.elasticsearch.common.settings.Settings;
 
+import java.util.Locale;
 import java.util.concurrent.CompletableFuture;
 
 import static io.crate.expression.reference.sys.check.AbstractSysCheck.CLUSTER_CHECK_LINK_PATTERN;
 import static io.crate.expression.reference.sys.check.AbstractSysCheck.getLinkedDescription;
 import static io.crate.expression.reference.sys.check.SysCheck.Severity.HIGH;
+import static io.crate.expression.reference.sys.check.SysCheck.Severity.LOW;
 import static io.crate.expression.reference.sys.check.SysCheck.Severity.MEDIUM;
 import static io.crate.license.LicenseExpiryNotification.MODERATE;
 
 @Singleton
-public class LicenseExpiryCheck implements SysCheck {
+public class LicenseCheck implements SysCheck {
 
     private static final int ID = 6;
-    private static final String LICENSE_NOT_CLOSE_TO_EXPIRY_DESCRIPTION = "Your CrateDB license is not close to expiry. Enjoy CrateDB!";
+    private static final String LICENSE_OK = "Your CrateDB license is fine. Enjoy CrateDB!";
     private static final String LICENSE_NA_COMMUNITY_DESCRIPTION = "CrateDB enterprise is not enabled. Enjoy the community edition!";
     private final boolean enterpriseEnabled;
+    private final ClusterService clusterService;
 
     private String description;
     private Severity severity = Severity.LOW;
     private final LicenseService licenseService;
 
     @Inject
-    public LicenseExpiryCheck(Settings settings, LicenseService licenseService) {
+    public LicenseCheck(Settings settings, LicenseService licenseService, ClusterService clusterService) {
         enterpriseEnabled = SharedSettings.ENTERPRISE_LICENSE_SETTING.setting().get(settings);
-        description = (enterpriseEnabled) ? LICENSE_NOT_CLOSE_TO_EXPIRY_DESCRIPTION : LICENSE_NA_COMMUNITY_DESCRIPTION;
+        this.clusterService = clusterService;
+        if (!enterpriseEnabled) {
+            description = LICENSE_NA_COMMUNITY_DESCRIPTION;
+        }
```

Comment:
as `description()` is not marked as `Nullable`, couldn't this change result in a NPE on the CE?